support/client/lib/vwf/view/blockly.js

```diff
@@ -100,7 +100,8 @@ define( [ "module", "vwf/view", "jquery", "vwf/model/blockly/JS-Interpreter/acor
                 self.kernel.setProperty( childID, "toolbox", self.options.toolbox );
                 Blockly.inject( document.getElementById( self.options.divName ), { 
                     path: this.options.blocklyPath,
-                    toolbox: document.getElementById( self.options.toolbox )
+                    toolbox: document.getElementById( self.options.toolbox ),
+                    "grid": { spacing: 40, length: 1, colour: '#ccc', snap: true }
```

Comment:
Yes, I tried to get it to work by specifying in index.config.yaml and making the appropriate changes on this side.. it didn't set properly. Since its not a primary task I didnt want to spend time debugging why config wasnt working for the moment.